docs/src/pages/demos/text-fields/ComposedTextField.tsx

```diff
@@ -17,88 +16,71 @@ const styles = (theme: Theme) =>
     formControl: {
       margin: theme.spacing(1),
     },
-  });
+  }),
+);
 
-export interface Props extends WithStyles<typeof styles> {}
+function ComposedTextField() {
+  const [labelWidth, setLabelWidth] = React.useState(0);
+  const [name, setName] = React.useState('Composed TextField');
+  const label = React.useRef<HTMLLabelElement>(null);
```

Comment:
```suggestion
  const labelRef = React.useRef<HTMLLabelElement>();
```
?

Reply:
The rename is fine, however it must be set to null as a default, because `InputLabel.ref` expects a ref of type `HTMLLabelElement | null`

Reply:
Correct :)